src/main/scala/spinoco/fs2/http/HttpServer.scala

```diff
@@ -36,7 +36,7 @@ object HttpServer {
     *                                     Request is not suplied if failure happened before request was constructed.
     *
     */
-  def apply[F[_] : ConcurrentEffect : Timer](
+  def apply[F[_] : ConcurrentEffect : Timer: ContextShift](
```

Comment:
Could we (and same for client) get rid of apply[F] as being constructor? Lets do mk, and have apply as summoner of the instance

Reply:
Changed them both to mk, however server cannot have summoner instance as the result is Stream[F, Unit].